gbfs.md

```diff
@@ -46,6 +46,7 @@ The table below lists the active systems Beryl maintains.
 | London        | London, GB                           | [https://gbfs.beryl.cc/v2/London/gbfs.json](https://gbfs.beryl.cc/v2/London/gbfs.json)               |
 | Norwich       | Norwich, GB                          | [https://gbfs.beryl.cc/v2/Norwich/gbfs.json](https://gbfs.beryl.cc/v2/Norwich/gbfs.json)             |
 | Watford       | Watford, GB                          | [https://gbfs.beryl.cc/v2/Watford/gbfs.json](https://gbfs.beryl.cc/v2/Watford/gbfs.json)             |
+| West Midlands       | West Midlands, GB                          | [https://gbfs.beryl.cc/v2/West_Midlands/gbfs.json](https://gbfs.beryl.cc/v2/West_Midlands/gbfs.json)             |
```

Comment:
```suggestion
| West Midlands | West Midlands, GB                    | [https://gbfs.beryl.cc/v2/West_Midlands/gbfs.json](https://gbfs.beryl.cc/v2/West_Midlands/gbfs.json) |
```

I know it doesn't change the output, but having the Markdown table line up just makes me happy 💅